modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -305,7 +309,37 @@ export default class NestedTabs extends Base {
 		}
 
 		if ( ! isActiveTab ) {
+			const isMobileVersion = 'none' === this.elements.$headingContainer.css( 'display' );
+
+			if ( isMobileVersion ) {
+				this.activateMobileTab( tabIndex );
+				return;
+			}
+
+			this.activateTab( tabIndex );
+		}
+	}
+
+	activateMobileTab( tabIndex ) {
+		// Timeout added to ensure that opening of the active tab starts after closing the other tab.
+		setTimeout( () => {
```

Comment:
@hein-obox Did Tzvi's suggestion to avoid `setTimeout(`) not work?

Reply:
@louiswol94 It didn't work.